README.md

```diff
@@ -170,15 +178,16 @@ class CreateQuestionHandler < RequestHandler::Base
     multipart do
       question do
         required true
+        type "json"
```

Comment:
wouldn't this change make the validation of the JSON payload below at line 207 fail since it's no longer attempting to extract the id, type, and attributes from the document?

Reply:
actually now it simply assumes that you send a "flat" json document without the jsonapi nesting

Reply:
yeah, but nilsding is right. the validation would fail... but i already changed it